src/syscall.c

```diff
@@ -23,23 +23,26 @@
  * system call: name, number
  */
 /* clang-format off */
-#define SUPPORTED_SYSCALLS           \
-    _(close,                57)      \
-    _(lseek,                62)      \
-    _(read,                 63)      \
-    _(write,                64)      \
-    _(fstat,                80)      \
-    _(exit,                 93)      \
-    _(gettimeofday,         169)     \
-    _(brk,                  214)     \
-    _(clock_gettime,        403)     \
-    _(open,                 1024)    \
-    IIF(RV32_HAS(SDL))(              \
-        _(draw_frame,       0xBEEF)  \
-        _(setup_queue,      0xC0DE)  \
-        _(submit_queue,     0xFEED)  \
-        _(setup_audio,      0xBABE)  \
-        _(control_audio,    0xD00D)  \
+#define SUPPORTED_SYSCALLS                 \
+    _(close,                57)            \
+    _(lseek,                62)            \
+    _(read,                 63)            \
+    _(write,                64)            \
+    _(fstat,                80)            \
+    _(exit,                 93)            \
+    _(gettimeofday,         169)           \
+    _(brk,                  214)           \
+    _(clock_gettime,        403)           \
+    _(open,                 1024)          \
+    _(sbi_base,             0x10)          \
```

Comment:
The SBI calls should be built conditionally. That is, specify `IIF(RV32_HAS(SYSTEM))`.